.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+Closes #
+
+<!-- 
+Thank you for contributing to ODK Central!
+
+Before sending this PR, please read
+https://github.com/getodk/central-frontend/blob/master/CONTRIBUTING.md
+-->
+
+#### What has been done to verify that this works as intended?
+
+#### Why is this the best possible solution? Were any other approaches considered?
+
+#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
+
+#### Does this change require updates to user documentation? If so, please file an issue [here](https://github.com/getodk/docs/issues/new) and include the link below.
+
+#### Before submitting this PR, please make sure you have:
+
+- [ ] run `npm run test` and `npm run lint` and confirmed all checks still pass OR confirm CircleCI build passes
+- [ ] verified that any code or assets from external sources are properly credited in comments
```

Comment:
I've noticed that this checkbox is often left unchecked. Is there a way to reword it to make it clear that in the common case in which there is no external code or assets, it's OK to check the box?

Reply:
😂 I usually delete this line because it feels wrong to check it, but also not applicable. It would need some kind of definitive "nothing external" clause for me to check it. 

`verified that any code or assets from external sources are properly credited in comments or that everything is internally sourced`

Reply:
Added!